helpers/lib/resourceHints.js

```diff
@@ -48,11 +48,11 @@ function addResourceHint(globals, path, rel, type, cors) {
 
     let index = globals.resourceHints.findIndex(({src}) => path === src);
     if (index >= 0) {
-        return;
+        return path;
     }
 
     if (globals.resourceHints.length >= resourceHintsLimit) {
-        return;
+        throw new Error(`Resource Hints limit (${resourceHintsLimit}) reached.`);
```

Comment:
Are we good to throw the error even it can be not under full user control in cases when cdn, stylesheet helpers are being used?

Reply:
`cdn` is not adding reload hint by default but `stylesheet` and `getFontsCollection`. But yeah, you are right, given that we are exposing adding hints as default for a couple of helpers, we better no to throw here but do nothing.